Improved Readme

jceelen opened the pull request:
Improved readability of lists by better indenting
Updated the procedure for **1.1 Default Apps Script Developer Console Project** to match with the current interface at Google.

jceelen committed:
Improved readabil README.md

jceelen committed:
Update README.md

jceelen committed:
Improved readability

lricoy commented on README.md:
```diff
@@ -31,47 +31,43 @@ You can do this one of two ways:
-1. Access the automatically created apps script Developer Console Project by
-  1. `Resources` > `Cloud Platform Project`
-  1. Click on the blue link at the top (`project-name - api-project-##########`) to access the correct Developer Console Project
+1. Go to the [Google Cloud Platform](https://console.cloud.google.com)
```
Could you also refer to the other option that is by accesing `Resources` > `Cloud Platform Project` please?

jceelen replied:
@lricoy: ure, but can you explain from which point you can access `Resources`? I do not see this in the UI in https://console.developers.google.com 
This is where I got lost in the first place :-)

jceelen replied:
@lricoy could you let me know how to proceed so I can update everything? Thanks! :-)

jceelen replied:
@lricoy nevermind, discovered it myself! I added som more improvements and think it is ready!

lricoy replied:
Hey @jceelen nice changes but I was referring to just entering `https://script.google.com/` and using the above menu of `Resources` > `Cloud Platform Project`.

lricoy replied:
Hey @jceelen nice changes but I was referring to just entering https://script.google.com/ and using the above menu of Resources > Cloud Platform Project.

lricoy commented on README.md:
```diff
@@ -31,47 +31,43 @@ You can do this one of two ways:
-  1. Finally, download your credentials using the `Download as JSON` button to the right.
-    * Save these credentials to a location of your choosing; `~/Downloads` is fine.
-  1. You may close the Developer Console window.
+   1. If you just enabled the Drive API, hit `Create Credentials`, else, go to `Credentials` in the left menu, select `Create credentials` and choose `OAuth client ID`
```
maybe `otherwise` instead of `else`?

jceelen replied:
Thanks, that's better!

lricoy reviewed (changes requested):
Really nice! Thanks @jceelen! Could you take at look the Google Cloud's  access step please? The other comment is totally optional :)

jceelen committed:
Update README.md

Refined procedure for 1.2 Independent Developer Console Project

jceelen committed:
Update README.md

Added some explanations to which method you should choose for getting Google Drive Credentials

lricoy commented:
Hey @jceelen. Sorry for the delay. Was out of town for the last week. I
will take a look tonight. I am glad you found it :)

Em Qua, 6 de dez de 2017 13:17, Jasper <notifications@github.com> escreveu:

> *@jceelen* commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/danthareja/node-google-apps-script/pull/75#discussion_r155220177>
> :
>
> > @@ -31,47 +31,43 @@ You can do this one of two ways:
>
>  #### 1.1 Default Apps Script Developer Console Project
>
> -1. Access the automatically created apps script Developer Console Project by
> -  1. `Resources` > `Cloud Platform Project`
> -  1. Click on the blue link at the top (`project-name - api-project-##########`) to access the correct Developer Console Project
> +1. Go to the [Google Cloud Platform](https://console.cloud.google.com)
>
> @lricoy <https://github.com/lricoy> nevermind, discovered it myself! I
> added som more improvements and think it is ready!
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/danthareja/node-google-apps-script/pull/75#discussion_r155220177>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB8Kv4tkW2p7UZf1Ce1b7O1b5ZH_TUppks5s9oW_gaJpZM4QweAe>
> .
>

lricoy commented on README.md:
```diff
@@ -23,55 +23,54 @@ Using **gapps**, you can develop your Apps Script locally and push files to the
 ## Quickstart
 
 ### 1. Get Google Drive Credentials
+To use gapps you need access to your files on your Google Drive through the API provided by Google. To make this happen you to create a `Developer Console Project`. 
```
[...] need to [...]

aurelienshz replied:
[...] you have to [...]

jceelen replied:
Fixed!

lricoy commented on README.md:
```diff
@@ -23,55 +23,54 @@ Using **gapps**, you can develop your Apps Script locally and push files to the
 ### 1. Get Google Drive Credentials
+To use gapps you need access to your files on your Google Drive through the API provided by Google. To make this happen you to create a `Developer Console Project`. 
+
+__The only purpose of this project is providing access, this is not the Google Script project you will be working on!__.
```
What is the intention here? I think I get it but looks a bit misleading IMHO. What do you mean with "it is not the project you will be working on"?

jceelen replied:
It was not clear to me what the purpose of creating this project was. I deleted it.

lricoy commented:
Hey @jceelen. Sorry again for the delay and thanks for the changes. Could you take a last look at the comments? Should not take long and then we could merge it :)

jceelen committed:
Update README.md

jceelen committed:
Update README.md

lricoy commented:
Thanks @jceelen 👍
